workshops/models.py

```diff
@@ -108,7 +108,7 @@ class Person(AbstractBaseUser, PermissionsMixin):
     email       = models.CharField(max_length=STR_LONG, unique=True, null=True, blank=True)
     gender      = models.CharField(max_length=1, choices=GENDER_CHOICES, null=True, blank=True)
     may_contact = models.BooleanField(default=True)
-    airport     = models.ForeignKey(Airport, null=True, blank=True)
+    airport     = models.ForeignKey(Airport, null=True, blank=True, on_delete=models.SET_NULL)
```

Comment:
On Thu, Jun 11, 2015 at 06:28:17AM -0700, Piotr Banaszkiewicz wrote:
> > -    airport     = models.ForeignKey(Airport, null=True, blank=True)
> > +    airport     = models.ForeignKey(Airport, null=True, blank=True, on_delete=models.SET_NULL)
> 
> I figured that we didn't need to forbid deletion of Airport if some
> Person links to it, because we don't manipulate airports at all.

If you don't delete airports, then leave this line alone and remove
the airport-deletion endpoint ;).

> I can change `on_delete` to `on_delete=models.PROTECT`, but this
> would raise a question: what would we do with Event.site and
> Event.organizer fields, that have `on_delete=models.SET_NULL`, too?

I'd use PROTECT there too, or stay with the default CASCADE if you're
feeling adventurous.  SET_NULL is for things like “Oops, not only does
that {link-target} not exist, but that {linker} doesn't even have a
{link-target}”.  That may be true, but I want it to be an explicit
decision for each linker, not something that happens automatically
after you've decided that a particular Site is a duplicate (in which
case Event.site should be updated to reference the deduplicated Site).
Why else would you be deleting Site/Airport/… that are referenced from
other fields besides deduping?